Src/libCZI/libCZI_Compositor.h

```diff
@@ -222,64 +222,107 @@ namespace libCZI
 
     public:
         /// Gets the tile composite of the specified plane and the specified ROI. The pixeltype is
-        /// determined by examing the first subblock found in the specified plane (which is an
+        /// determined by examining the first subblock found in the specified plane (which is an
         /// arbitrary subblock). A newly allocated bitmap is returned.
         ///
-        /// \param  roi             The ROI (given in _raw-subblock-coordinate-system_, c.f. @ref coordinatesystems).
+        /// \param  roi             The ROI and the coordinate system it is defined in.
         /// \param  planeCoordinate The plane coordinate.
         /// \param  pOptions        Options for controlling the operation.
         ///
         /// \returns    A std::shared_ptr<libCZI::IBitmapData> containing the tile-composite.
-        ///
-        /// ### remarks It needs to be defined what is supposed to happen if there is no subblock found
-        ///     in the specified plane.
-        virtual std::shared_ptr<libCZI::IBitmapData> Get(const libCZI::IntRect& roi, const IDimCoordinate* planeCoordinate, const Options* pOptions) = 0;
+        virtual std::shared_ptr<libCZI::IBitmapData> Get(const libCZI::IntRectAndFrameOfReference& roi, const IDimCoordinate* planeCoordinate, const Options* pOptions) = 0;
```

Comment:
...ehm, no, or I am not really convinced this is a good idea. First of all, those are no "getters" in the [usual sense](https://www.geeksforgeeks.org/cpp-getters-and-setters/), those are "quite massive operations with the unfortunate? name Get". Then... I am not sure whether they are really technically "const methods" now, or there could be very subtle mutations going on (e.g. technically, grabbing a critical-section is already a mutation). 
Whether this applies or (if it is actually debatable whether the operations are currently really "const"), ja, would have to be checked. But yeah, I think it could make sense here - but not as part of this PR, please understand.